src/rust/engine/options/src/args.rs

```diff
@@ -77,6 +79,31 @@ impl Args {
         }
         Ok(None)
     }
+
+    fn get_list<T>(
```

Comment:
A generic variant of the old get_string_list, which is now reimplemented using this (as are the other scalar list types). The parse_list argument is the specific parser function we should apply to the raw string to yield the needed list type.